security/access_token.rst

```diff
@@ -91,7 +91,7 @@ This handler must implement
         ) {
         }
 
-        public function getUserBadgeFrom(string $accessToken): UserBadge
+        public function getUserBadgeFrom(string $token): UserBadge
```

Comment:
are you sure? see https://github.com/symfony/symfony/blob/94d6bbbc7691789b9cc946ef1b2964ea3180cecf/src/Symfony/Component/Security/Http/AccessToken/AccessTokenHandlerInterface.php#L28

Reply:
Change to $accessToken to comply with the interface.